src/Concerns/WithClientAutoInitializeTrait.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Liior\SymfonyTestHelpers\Concerns;
+
+use Liior\SymfonyTestHelpers\Exception\ClientNotCreatedException;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\BrowserKit\AbstractBrowser;
+
+trait WithClientAutoInitializeTrait
+{
+    protected function setUp(): void
+    {
+        if (\method_exists($this, 'initializeClient')) {
+            $this->initializeClient();
+        }
+    }
+
+    protected function tearDown(): void
+    {
+        if (isset($this->client)) {
+            $this->client = null;
+        }
```

Comment:
We have to add this line here in order to fix a bug I just saw :
`static::ensureKernelShutdown();`

If we don't have this line, an Exception is thrown by the WebTestCase base class (see Symfony\Bundle\FrameworkBundle\Test\WebTestCase on line 44)